TestSuite/test_tools.py

```diff
@@ -35,3 +35,35 @@ def __enter__(self):
 
     def __exit__(self, *args):
         os.chdir(self.current)
+
+
+def str_in_call_args_list(call_args_list, required_str):
+    """
+    Checks whether required_str is in any of the call_args in call_args_list
+    Args:
+        call_args_list: From a mocker
+        required_str: String to search in any of the call_args_list
+    :return: True is required_str was found, False otherwise
+    """
+    for current_call in call_args_list:
+        if current_call and isinstance(current_call[0], tuple):
+            if required_str in current_call[0][0]:
+                return True
+    print(f"Could not find {required_str=}")
```

Comment:
remove print

Reply:
It's needed for debugging (when running in the build, etc.)